charms/kfp-profile-controller/files/upstream/sync.py

```diff
@@ -39,13 +39,22 @@ def emit_settings_to_logs(settings):
     logger.info(f"Settings = {safe_settings}")
 
 
-def get_settings_from_env(controller_port=None,
-                          visualization_server_image=None, frontend_image=None,
-                          visualization_server_tag=None, frontend_tag=None,
-                          disable_istio_sidecar=None, minio_access_key=None, minio_secret_key=None,
-                          minio_host=None, minio_port=None, minio_namespace=None,
-                          kfp_default_pipeline_root=None, metadata_grpc_service_host=None,
-                          metadata_grpc_service_port=None):
+def get_settings_from_env(
```

Comment:
Are the changes in this file just to satisfy the linting/formatting, or are there substanttitive changes here too?  This file comes from upstream, so to make our lives easier when reconciling changes with upstream in the future we should avoid formatting changes here.  If this was to satisfy out code checks, we should add an exemption for this file

Reply:
It was to satisfy black lint requirements. I will revert the changes and add the upstream file to exceptions.